### PR TITLE
Feature/gravatar profile

### PR DESCRIPTION
**Pull request by dividezero**
### Description

The gravatar profile picture. New pull request because wanted to remove email =p

### Issues fixed

Redid the gravatar profile according to the comments in the last pull request

### Checklist

- [x] `npm test` returns no warnings or errors.

### CODE REVIEW

File: src/components/GravatarProfile/GravatarProfile.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import md5 from 'md5';
+
+import './GravatarProfile.css';
+
+const GravatarProfile = ({email, gravStyle}) => (
+	<div className='GravatarProfile'>
+		<img src={'https://www.gravatar.com/avatar/' + md5(email.trim().toLowerCase()) + '?d=' + gravStyle} />
```
Comment (ShaunBulbrook) [inline src/components/GravatarProfile/GravatarProfile.js]: alt attribute is required on the element (however it can just be an empty string)

Comment (dan1elhughes) [reply src/components/GravatarProfile/GravatarProfile.js]: Specifically i think it _should_ be an empty string for accessibility reasons, right? Going off something @peasandwell mentioned

Comment (dividezero) [reply src/components/GravatarProfile/GravatarProfile.js]: noted. ive added an empty alt text

File: src/components/GravatarProfile/GravatarProfile.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+GravatarProfile.description = `
+Used as a visualisation of a customers Gravatar profile picture.
+`;
+
+GravatarProfile.propTypes = {
+	/** email to show the profile of the customer */
+	email: PropTypes.string.isRequired,
+	/** gravatar style to be used to generate default gravatars. Options are mm, identicon, monsterid, wavatar, retro, robohash */
+	gravStyle: PropTypes.string,
```
Comment (dan1elhughes) [inline src/components/GravatarProfile/GravatarProfile.js]: I know I'm late on this PR, but can we `PropTypes.oneOf([...])` this?

Comment (dan1elhughes) [reply src/components/GravatarProfile/GravatarProfile.js]: In another PR

**Pull request merged**